From 021fe33d58902608d8439927c2a078712ab0265e Mon Sep 17 00:00:00 2001
From: auto commit <unknown@unknown>
Date: Fri, 24 Oct 2008 12:23:08 -0700
Subject: [PATCH] auto_msg: importing 0001-cavium-hook-trace-buffer.patch

This is an automatic import of patch 0001-cavium-hook-trace-buffer.patch, no headers were
detected and a default message was constructed
---
 arch/mips/cavium-octeon/Makefile |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/mips/cavium-octeon/Makefile b/arch/mips/cavium-octeon/Makefile
index 8c29038..c15c23f 100644
--- a/arch/mips/cavium-octeon/Makefile
+++ b/arch/mips/cavium-octeon/Makefile
@@ -25,6 +25,7 @@ obj-$(CONFIG_PCI)                     += pci.o pci_chips.o i8259.o
 obj-$(CONFIG_PCI_MSI)                 += msi.o
 obj-$(CONFIG_PCI)                     += pcie.o
 obj-$(CONFIG_CAVIUM_OCTEON_WATCHDOG)  += watchdog.o watchdog_nmi.o
+obj-$(CONFIG_CAVIUM_OCTEON_TRA)       += octeon-tra.o
 
 clean:
 	rm -f *.o
-- 
1.5.5.1

